lib/timeline/Timeline.js

```diff
@@ -120,15 +120,27 @@ function Timeline (container, items, groups, options) {
   this.itemsData = null;      // DataSet
   this.groupsData = null;     // DataSet
 
-  this.on('tap', function (event) {
+  this.dom.root.onclick = function (event) {
     me.emit('click', me.getEventProperties(event))
-  });
-  this.on('doubletap', function (event) {
+  };
+  this.dom.root.ondblclick = function (event) {
     me.emit('doubleClick', me.getEventProperties(event))
-  });
+  };
   this.dom.root.oncontextmenu = function (event) {
     me.emit('contextmenu', me.getEventProperties(event))
   };
+  this.dom.root.onmousedown = function (event) {
+    me.emit('mouseDown', me.getEventProperties(event))
+  };
+  this.dom.root.onmouseover = function (event) {
+    me.emit('mouseOver', me.getEventProperties(event))
+  };
+  this.dom.root.onmouseup = function (event) {
+    me.emit('mouseUp', me.getEventProperties(event))
+  };
+  this.dom.root.onmousemove = function (event) {
```

Comment:
do we really want to emit this event. This happens quite often? What is the usecase for this?